content/warnings/invalid-hook-call-warning.md

```diff
@@ -43,13 +43,13 @@ function useWindowWidth() {
 }
 ```
 
-To avoid confusion, it’s **not** supported to call Hooks in other cases:
+Untuk menghindari kebingungan, Pemanggilan Hooks tidak didukung pada kasus-kasus berikut:
 
-* 🔴 Do not call Hooks in class components.
-* 🔴 Do not call in event handlers.
-* 🔴 Do not call Hooks inside functions passed to `useMemo`, `useReducer`, or `useEffect`.
+* 🔴 Jangan panggil Hooks di dalam komponen berbasi kelas.
+* 🔴 Jangan panggil Hooks di penangan event.
```

Comment:
```suggestion
* 🔴 Jangan panggil Hooks di *event handler*.
```

*Event handler* tidak perlu diterjemahkan, as per the [glossary](https://github.com/reactjs/id.reactjs.org/wiki/Glosarium#e).